packages/ckeditor5-easy-image/README.md

```diff
@@ -1,4 +1,4 @@
-CKEditor 5 Easy Image with Cloud Services
+CKEditor 5 easy image feature with Cloud Services
```

Comment:
What about other places, where feature name is shown? For example, in this file a few line below there is the following sentence:

> This package implements the [Easy Image](https://ckeditor.com/docs/ckeditor5/latest/features/easy-image.html) feature for CKEditor 5.

Shouldn't that also be changed?

Similar phrases also appear in other files.

Reply:
That's a nice catch, I would think that you're correct. I will review the files again and see what else I'll find.

Reply:
How about Easy Image being a brand name? This is not just a feature, this is kinda sales point. Like Revision History or smthn?